pl/thunderbird/start/release.lang

```diff
@@ -7,43 +7,43 @@ Witaj w <span>Thunderbirdzie</span>
-You might already know that Thunderbird improvements are no longer paid for by Mozilla. Fortunately there is an active community keeping it running and developing it further. But to survive long term, the project needs funding.
+Może już wiesz, że Mozilla nie płaci już za rozwój Thunderbirda. Na szczęście istnieje aktywna społeczność nadal go rozwijająca. Mimo to aby przetrwać w dłuższym okresie, projekt potrzebuje pieniędzy.
 
 
 ;Thunderbird is currently transitioning to an independent organization. Being independent, we can shape our own fate, but there is significant infrastructure that must be maintained to deliver the application to our tens of millions of users. For Thunderbird to survive and continue to evolve, we need your support and ask for your donation today. All the money donated will go directly to funding Thunderbird development and infrastructure.
-Thunderbird is currently transitioning to an independent organization. Being independent, we can shape our own fate, but there is significant infrastructure that must be maintained to deliver the application to our tens of millions of users. For Thunderbird to survive and continue to evolve, we need your support and ask for your donation today. All the money donated will go directly to funding Thunderbird development and infrastructure.
+Thunderbird jest obecnie w trakcie przeprowadzki do niezależnej organizacji. Niezależność umożliwia nam samodzielne kształtowanie przyszłości, ale udostępnianie Thunderbirda dziesiątkom milionów użytkowników wymaga znacznej infrastruktury. Potrzebujemy Twojego wsparcia finansowego, aby Thunderbird mógł przetrwać i nadal się rozwijać. Wszystkie zebrane fundusze będą przeznaczone wyłącznie do opłacenia rozwoju i infrastruktury Thunderbirda.
 
 
 ;Donate to Thunderbird
-Donate to Thunderbird
+Wspomóż Thunderbirda
 
 
 ;Thunderbird is the leading open source, cross-platform email and calendaring client, free for business and personal use. We want it to stay secure and become even better.
-Thunderbird is the leading open source, cross-platform email and calendaring client, free for business and personal use. We want it to stay secure and become even better.
+Thunderbird to wiodący międzyplatformowy klient poczty i kalendarza open source, darmowy dla przedsiębiorstw i użytku osobistego. Chcemy, aby nadal był bezpieczny i stał się jeszcze lepszy.
 
 
 ;If you like Thunderbird, please consider a donation!
-If you like Thunderbird, please consider a donation!
+Jeśli lubisz Thunderbirda, rozważ wpłacenie darowizny!
 
 
 ;By donating, you will help us to continue delivering an ad-free top-notch email client.
-By donating, you will help us to continue delivering an ad-free top-notch email client.
+Wpłacając, pomożesz nam nadal dostarczać najlepszego klienta poczty, całkowicie wolnego od reklam.
 
 
 ;Make a donation
-Make a donation
+Wpłać darowiznę
```

Comment:
I don't think this is correct ("darowizny"), see also http://bugs.aviary.pl/show_bug.cgi?id=4981